tasks/setup.yml

```
@@ -0,0 +1,7 @@
+---
+# Perform necessary setup
+- name: Install apt-transport-https to support HTTPS repository URLs
+  ansible.builtin.apt:
+    force_apt_get: yes
+    name: apt-transport-https
+    update_cache: yes
```

I think you can remove two unnecessary lines.
```suggestion
  ansible.builtin.apt:
    name: apt-transport-https
```

---

Another facepalm. I'm on fire today.

---

_En fuego_, we call it here at Planet Piss.

---

The removal of `update_cache` actually raises an important (to me) point. I think we only get away with this because in our molecule prepare step we use [cisagov/ansible-role-upgrade](https://github.com/cisagov/ansible-role-upgrade), however we do not make that a requirement for the role itself. I'm leery of a scenario where this is used without that step.

---

You're right, but at some point we opted for doing things this way.  The alternative is to add a cache update before the first package is installed on all Ansible roles.  The cache update would also have to be tagged appropriately to avoid idempotence woes.

---

I'll align with how we currently do things. I guess my concern is we make an Ansible role useful to not us and it blows up in someone's face unexpectedly.